modules/network-observability-loki-install.adoc

```diff
@@ -34,25 +34,26 @@ There are several ways you can install Loki. One way you can install the Loki Op
 +
 . Create a `Secret` YAML file. You can create this secret in the web console or CLI.
 .. Using the web console, navigate to the *Project* -> *All Projects* dropdown and select *Create Project*. Name the project `netobserv` and click *Create*.
-.. Navigate to the Import icon, *+*, in the top right corner. Drop your YAML file into the editor. It is important to create this YAML file in the `netobserv` namespace that uses the `access_key_id` and `access_key_secret` to specify your credentials. 
+.. Navigate to the Import icon ,*+*, in the top right corner. Drop your YAML file into the editor. 
```

Comment:
```suggestion
.. Navigate to the Import icon, *+*, in the top right corner. Drop your YAML file into the editor. 
```

Is this space on purpose ? :thinking: :eye:

Reply:
Nope, the space should be as you've suggested. Good 👁️

Reply:
Addressed